src/NServiceBus.Transport.RabbitMQ/Routing/DirectRoutingTopology.cs

```diff
@@ -129,11 +154,11 @@ void CreateExchange(IModel channel, string exchangeName)
 
             try
             {
-                channel.ExchangeDeclare(exchangeName, ExchangeType.Topic, useDurableExchanges);
+                channel.ExchangeDeclare(exchangeName, ExchangeType.Topic, useDurableEntities);
             }
             catch (Exception)
             {
-
+                // TODO: Any better way to make this idempotent?
```

Comment:
FYI this has just been copied over from the conventional topology which contains this comment